lib/agents/reviewer.tsx

```diff
@@ -1,7 +1,9 @@
 import {initializeAgentExecutorWithOptions} from 'langchain/agents'
 import {ChatOpenAI} from 'langchain/chat_models/openai'
 import {SerpAPI} from 'langchain/tools'
+import parse, {Change, Chunk, File} from 'parse-diff'
 import env from '~/env.mjs'
+import {codeComment} from '~/tools/codeComment'
```

Comment:
YO THIS REVIEWING FEATURE IS SICK

@arihanv flagging this (as a much higher-level PR review than usual, which is SICK):

Since `codeComment` is used below, how could we prevent this kind of error? Eg. prompt engineering ("this snippet is not the full file and might be missing parts etc") or passing longer code snippets?